cpp/core/CMakeLists.txt

```diff
@@ -141,7 +141,13 @@ if(USE_AVX512)
 endif()
 
 # Set up Proto
-set(PROTO_OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/proto")
+if (NOT DEFINED VELOX_HOME)
```

Comment:
This code is duplicated with https://github.com/oap-project/gluten/blob/main/cpp/velox/CMakeLists.txt#L40

And core directory is used for common code, can we move it to velox CmakeLists file?

Reply:
Thanks, Just Updated.